server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -21,6 +21,23 @@ import {
 	TelemetryContextHeaderName,
 } from "./constants";
 import { getGlobalTimeoutContext } from "./timeoutContext";
+import { isAxiosCanceledError } from "./utils";
+
+/**
+ * @internal
+ */
+export function setupAxiosInterceptorsForAbortSignals(
+	getAbortController: () => AbortController | undefined,
```

Comment:
Maybe I understand this part wrong, does this means:
`() => (AbortController | undefined),`
Or `(() => AbortController) || undefined`?

Is there a way to specify this?

Reply:
It's the 1st one, `() => (AbortController | undefined)`

Reply:
> Is there a way to specify this?

Parenthesis are unnecessary for the first example, `() => (AbortController | undefined)`, but would be necessary to specify your second example, `(() => AbortController) | undefined` (note: `|` not `||` because `||` is not a valid Union Type notation).

So, to answer your question, yes, but the way it is right now specifies it correctly.

Reply:
I would rather keep parenthesis for readability.